helpers/tools.ts

```diff
@@ -188,6 +188,35 @@ For better results, you can specify the region parameter to get results from a s
       },
     ],
   },
+  {
+    display: "Azure Code Interpreter",
+    name: "azure_code_interpreter.AzureCodeInterpreterToolSpec",
+    supportedFrameworks: ["fastapi"],
+    type: ToolType.LLAMAHUB,
+    config: {
+      // TODO: `managment` is a typo from the llamahub code, fix it in the future
+      pool_managment_endpoint:
+        "Please follow this guideline to create and get the pool management endpoint: https://learn.microsoft.com/azure/container-apps/sessions?tabs=azure-cli",
+    },
+    dependencies: [
+      {
+        name: "llama-index-tools-azure-code-interpreter",
+        version: "0.1.1",
```

Comment:
@manekinekko Great. Now you'll just need to update this version once https://github.com/run-llama/llama_index/pull/14732 is released and the setup should be easier :D